### PR TITLE
Added a few images

### PR DESCRIPTION
**Pull request by Shaurya50211**
I added images for clear view

### CODE REVIEW

**Comment by ShehrozIrfan** [comment]
@Shaurya50211 
Great!
Your PR is merged.
You can follow me and star this repository for further details.
Have fun!

File: README.md [hunk 1]
```diff
@@ -33,5 +35,6 @@ Copy the code already present, paste it and make the following changes.
 
 ### OUR VALUABLE CONTRIBUTORS✨
 <a href="https://github.com/ShehrozIrfan/hacktoberfest-2022-first-contribution/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=ShehrozIrfan/hacktoberfest-2022-first-contribution" />
+  <img src="https://contrib.rocks/image?repo=ShehrozIrf![Uploading image.png…]()
```
Comment (ShehrozIrfan) [inline README.md:38]: @Shaurya50211 
I think it won't work. 
Fix it.

Comment (Shaurya50211) [reply README.md:38]: Should I remove those images?

Comment (ShehrozIrfan) [reply README.md:38]: No. Don't remove.
I merged your changes, but not unable to see any changes.
You can take the latest pull from master. And check is it working alright?

**Comment by ShehrozIrfan** [comment]
Hey there!
If your contribution is excluded for this repository, then please visit the repository [Ruby Rocks](https://github.com/ShehrozIrfan/ruby-rocks/) for contribution to hacktoberfest. It contains a plenty of beginner's friendly issues accepted for hacktoberfest.

Have fun!

**Comment by Shaurya50211** [comment]
It got accepted

**Pull request merged**